docs/builders/arm.mdx

```diff
@@ -526,27 +526,6 @@ prevent running afoul of Azure decency controls.
 The password alphabet used for random values is
 **0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ**.
 
-### Deprecation Warning
```

Comment:
Will post a longer clarification on the removal of the deprecation warning for VHDs in the release notes for v2.0.0